docs/rules/REMEDIATION_TIER_REPORT.md

```
@@ -0,0 +1,210 @@
+| L049 | low | task | Rename loop var to item_ prefix |
+| L050 | low | task | Rename variable to lowercase_underscore |
+| L051 | low | task | Normalize Jinja spacing |
+| L061 | low | task | Normalize yes/no/True/False to true/false |
+| L062 | low | task | Convert key=value to YAML mapping style |
+| L063 | low | block | Add block name |
+| L064 | low | task | Replace meta end_play with end_host |
+| L073 | low | task | Fix YAML indentation to 2 spaces |
+| L076 | low | task | Rewrite ansible_* to ansible_facts[] |
+| L078 | low | task | Convert dot to bracket notation in Jinja |
+| L080 | low | task | Prefix internal vars with underscore |
+| L091 | low | task | Add | bool filter to bare when vars |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)